docs/getting-started/uninstall_turtles_operator.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 5
+---
+
+# Uninstall Rancher Turtles Operator
+
+This gives an overview of Rancher Turtles Operator uninstallation process.
+
+:::caution
+When installing Rancher Turtles in your Rancher environment, by default, Rancher Turtles enables the Cluster API Operator cleanup. This includes cleaning up Cluster API Operator specific webhooks and deployments that otherwise would block the uninstallation.
```

Comment:
```suggestion
When installing Rancher Turtles in your Rancher environment, by default, Rancher Turtles enables the Cluster API Operator cleanup. This includes cleaning up Cluster API Operator specific webhooks and deployments that otherwise cause issues with Rancher provisioning.
```